### ISSUE TITLE
Incompatibility with any mod including MixinExtras 0.2.0

### ISSUE THREAD

**Issue by Wesley1808** [Thelnfamous1/Combat-Next#8]
Combat-Next requests a version dependency on MixinExtras that doesn't seem to include newer versions of the library,
which breaks it with any other mod that includes these versions https://github.com/Wesley1808/ServerCore/issues/83
```
[11:01:10] [main/ERROR] [ne.mi.ja.se.JarSelector/]: Failed to select jars for [ResolutionFailureInformation{failureReason=VERSION_RESOLUTION_FAILED, 

identifier=ContainedJarIdentifier[group=com.github.LlamaLad7, artifact=MixinExtras],
sources=[
SourceWithRequestedVersionRange{source=[Mod File: mixinextras-forge-0.2.0.jar], requestedVersionRange=[0.2.0,), includedVersion=0.2.0},
SourceWithRequestedVersionRange{source=[Mod File: combat_next-1.20.1-1.0.1-all.jar], requestedVersionRange=[0.1.1,0.2), includedVersion=0.1.1}
]}]
```

**Comment by Thelnfamous1** [comment]
Fixed in 1.20.1-1.0.2

**Comment by TrkvonStpd1e11** [comment]
Can the fixed feature be ported back to 1.19.2? 1.19.2 also has such problem.
